tox.ini

```diff
@@ -22,9 +22,9 @@ commands =
 envdir = .tox/lint
 basepython = python3
 commands =
-  black --check {posargs} qiskit test tools
+  black --check {posargs} qiskit test tools examples
```

Comment:
Should you add setup.py here too? Also what about the `black` target?

Reply:
👍